lib/config/folderconfiguration.go

```diff
@@ -235,6 +239,14 @@ func (f *FolderConfiguration) prepare() {
 	if f.MarkerName == "" {
 		f.MarkerName = DefaultMarkerName
 	}
+
+	if f.ModTimeWindowS < 2 && f.Path != "" {
```

Comment:
Should this be ModTimeWindowS <= 0 ?
1s is still valid, for ext2 et al.

Reply:
Then we'd need that case for ext2. Right now my intention is to force at least 2s window if we detect a fat filesystem. If the window is already >=2s, there's no need to check.

Reply:
Ok, got that.

However I just "remembered" something else: Shouldn't we leave it at 0 regardless of FS (fat or anything else) for all OSes except android? AFAIK the problem isn't the precision, the mtimefs takes care of that, but the weird mtime caching thingy of android that resets on reboot, right?

Reply:
I think we should check for GOOS=android, check the path of the folder, if it's in /storage/0 or something like that, and currently stored value is zero (negative would mean do not guess), we return 2 instead of zero.

Reply:
And yes, you are right that mtime fs fixes that on other platforms, but I think it's ok to have this as a setting, in case we want to drop mtime fs at some point.

Reply:
So /storage/0 definitely isn't it. For me it's e.g. `/storage/CB88-16EA` for external SD. And internal SD (`/storage/emulated` for me) is some android hackery remount thingy, the base FS isn't FAT, no idea how it behaves though and whether the 2s windows is necessary. So I propose to just put the 2s window an android entirely - fix a dirty problem with a dirty fix.

Reply:
Hi, putting it entirely is okay for me at a first glance. It would be more exact if we call mount | grep fat (or bufferedreader string compares if grep isnt available) and then only set this Attribut forcefully through the wrapper if a folder path points to sdcard. Source : https://stackoverflow.com/questions/27227198/how-do-i-check-the-type-of-sdcards-filesystem
Even more better, we could do it in java (Environment.getExternalStorageDirectory()) and compare that to the folder path for deciding 0 or 2 secs.